### PR TITLE
Switch to oxrs libs

### PR DESCRIPTION
**Pull request by sumnerboy12**
I have removed the sensors lib initially, just to get everything working. Once proven and merged you can add that back in, should be pretty straightforward.

I have only compiled and tested on D1 and LilyGO (and haven't actually tested the outputs, just that it boots and connects to MQTT etc).

When you get time run thru each h/w type and check this new code works.

This will be a lot easier to maintain, and is using standard OXRS code without any duplication.

### CODE REVIEW

**Comment by moinmoin-sh** [comment]
The screen function is broken.
- no port displayed
- no event/bottom line
- header , network and TEMP is shown

**Comment by sumnerboy12** [comment]
Thanks for testing @moinmoin-sh. It was a simple typo which I have now fixed (and tested locally on a spare Rack32). All working now.

File: src/main.cpp [hunk 1]
```diff
@@ -360,8 +362,7 @@ void processStrips(uint8_t controller)
 
   #if defined(OXRS_RACK32)
   // update port visualisation on screen
-  OXRS_LCD* screen = oxrs.getLCD();
-  screen->process(controller, pcaState);
+  oxrs.getLCD()->process(controller, pcaState);
```
Comment (moinmoin-sh) [inline src/main.cpp]: you might consider using here the existing API call instead
`oxrs.updateDisplayPorts(controller, pcaState);`

Comment (sumnerboy12) [reply src/main.cpp]: Thanks Moin - done.

**Review by austinscreations — APPROVED** [review]

**Pull request merged**